bc_obps/registration/tests/endpoints/_operations/test_operation_id_endpoints.py

```diff
@@ -0,0 +1,738 @@
+        # test setup
+        operator = operator_baker()
+        operation = operation_baker()
+        operation.operator_id = operator.id
+        operation.save(update_fields=['operator_id'])
+        TestUtils.authorize_current_user_as_operator_user(self, operator)
+        original_contact_id = operation.point_of_contact.id
+
+        # update fields visible in section 1 of form (on frontend)
+        update_from_form_section_1 = OperationUpdateIn(
+            name='New and Improved Legal Name',
+            type='Type A',
+            operator_id=operator.id,
+            documents=[],
+            regulated_products=[],
+            naics_code_id=operation.naics_code_id
+            # reporting_activities=[1],
+        )
+        put_response_1 = TestUtils.mock_put_with_auth_role(
+            self,
+            'industry_user',
+            self.content_type,
+            update_from_form_section_1.model_dump_json(),
+            custom_reverse_lazy("update_operation", kwargs={"operation_id": operation.id})
+            + "?submit=false&form_section=1",
+        )
+        assert put_response_1.status_code == 200
+        assert Operation.objects.count() == 1
+        retrieved_op = Operation.objects.first()
+        assert retrieved_op.name == 'New and Improved Legal Name'
+        assert retrieved_op.type == 'Type A'
+
+        # update fields visible in section 2 of form
+        update_from_form_section_2 = OperationUpdateIn(
+            name='This name should not be changed',
+            type='bad type',
+            operator_id=operator.id,
+            naics_code_id=operation.naics_code_id,
+            regulated_products=[],
+            # reporting_activities=[],
+            is_external_point_of_contact=True,
+            external_point_of_contact_first_name='Bart',
+            external_point_of_contact_last_name='Simpson',
+            external_point_of_contact_position_title='Scoundrel',
+            external_point_of_contact_email='bart@email.com',
+            external_point_of_contact_phone_number='+17787777777',
+        )
+        put_response_2 = TestUtils.mock_put_with_auth_role(
+            self,
+            'industry_user',
+            self.content_type,
+            update_from_form_section_2.model_dump_json(),
+            custom_reverse_lazy("update_operation", kwargs={"operation_id": operation.id})
+            + "?submit=false&form_section=2",
+        )
+        assert put_response_2.status_code == 200
+        assert Operation.objects.count() == 1
+        retrieved_op = Operation.objects.first()
+        # should be 1 contacts - we updated the existing point_of_contact
+        assert Contact.objects.count() == 1
+        assert retrieved_op.point_of_contact.first_name == 'Bart'
+        assert retrieved_op.point_of_contact_id == original_contact_id
+        assert retrieved_op.name == 'New and Improved Legal Name'
+
+    def test_only_approved_industry_users_can_update_operations(self):
```

Comment:
This test was previously both a create and update test; I split it to match the new endpoint structure. None of the other tests have been changed, only moved to different files.